### DISCUSSION TITLE
Checkout a quantity of consumable

### DISCUSSION THREAD

**Discussion by thiago-gil** [snipe/snipe-it#10327] [Ideas]
It would be very useful, if we could checkout more than 1 consumable at once, instead of only 1 each

**Comment by Hamartia123** [comment]
I believe this is a function they have listed as in development for Snipe-IT v6
https://github.com/snipe/snipe-it/issues/9834

**Comment by snakebag** [comment]
We are on Ver6.1 and I am still not seeing a way to check out multiple quantities of consumables.
